index.html

```diff
@@ -2,237 +2,257 @@
-            <h3 class="social order2">LET'S CONNECT</h3>
-            <ul class="social-links">
-              <li><a href="#"><i class="fa-brands fa-twitter li1"></i></a></li>
-              <li><a href="#"><i class="fa-brands fa-linkedin li2"></i></a></li>
-              <li><a href="#"><i class="fa-brands fa-m li3"></i></a></li>
-              <li><a href="#"><i class="fa-brands fa-github li4"></i></a></li>
-              <li><a href="#"><i class="fa-brands fa-angellist li5"></i></a></li>
-            </ul>
-            <button class="resume">Get my resume</button>
-          </div>
-          <div class="skills-list">
-            <div class="languages">
-              <div class="container">
-                <span class="lang">Languages</span>
-                <i class="fa-solid fa-angle-down btn1"></i>
-              </div>
-              <div class="language-list">
-                <ul>
-                  <li class="l-list"><img class="images" src="image/EllipseJS.png"><span
-                      class="js">Javascript</span></li>
-                  <li class="l-list"><img class="images" src="image/EllipseHTML.png"><span
-                      class="js">HTML</span></li>
-                  <li class="l-list"><img class="images" src="image/EllipseCSS.png"><span class="js">CSS
-                    </span></li>
-                </ul>
-              </div>
-            </div>
-            <div class="container">
-              <span class="lang">Frameworks</span>
-              <i class="fa-solid fa-angle-right btn1"></i>
-            </div>
-            <div class="container">
-              <span class="lang">Skills</span>
-              <i class="fa-solid fa-angle-right btn1"></i>
-            </div>
-          </div>
+        <div class="container">
+          <span class="lang">Frameworks</span>
+          <i class="fa-solid fa-angle-right btn1"></i>
+        </div>
+        <div class="container">
+          <span class="lang">Skills</span>
+          <i class="fa-solid fa-angle-right btn1"></i>
         </div>
-      </section>
-  
+      </div>
+    </div>
+  </section>
+  <section class="contact-form">
+    <h1>Contact me</h1>
+    <p>If you have an application you are interested in developing, a feature that you need built or a project that
+      needs coding. I’d love to help with it </p>
+    <form action="https://formspree.io/f/{form_id}" method="post">
+      <ul>
+        <li> <input type="text" placeholder=" name" name="user_name" maxlength="30" required> <br>
+        </li>
+        <li>
+          <input type="email" placeholder=" email" name="user_email" required><br>
+        </li>
+        <li>
+          <textarea name="user_message" id="message" cols="30" rows="10" placeholder=" Write your message here"
+            maxlength="500" required></textarea><br>
+        </li>
+        <li>
+          <input type="submit" value="Get in touch">
+        </li>
+      </ul>
+
+
+    </form>
+  </section>
```

Comment:
- [ ] While you did wonderful job settings all the elements inside the page :+1: , it is required to use flexbox to layout the elements. You can find more information about flexbox [here](https://css-tricks.com/snippets/css/a-guide-to-flexbox/).